examples/flutter_login/lib/app.dart

```diff
@@ -17,14 +17,25 @@ class App extends StatelessWidget {
   final AuthenticationRepository authenticationRepository;
   final UserRepository userRepository;
 
+  @override
+  State<App> createState() => _AppState();
+}
+
+class _AppState extends State<App> {
+  @override
+  void dispose() {
+    widget.authenticationRepository.dispose();
```

Comment:
Thanks for the pull request! Ideally I think the rule of thumb should be whoever creates the object should be responsible for disposing the object. In this case, the App does not create the authentication repository instance so it should not be responsible for disposing it. Can we move the disposable up to where the repository was created? Thanks!

Reply:
Oh! Sure 
Can move it.. thanks